config/application.rb

```diff
@@ -1,6 +1,9 @@
 require_relative 'boot'
 
-require 'rails/all'
+# Select only parts we need from rails/all
+require 'rails'
+require 'active_record/railtie'
+require 'action_controller/railtie'
```

Comment:
it is my hope that the tests will run faster if they don't need to load so much stuff, most of which we weren't using!